plugin/evm/client.go

```diff
@@ -72,3 +75,9 @@ func (c *client) GetVMConfig(ctx context.Context) (*Config, error) {
 	err := c.requester.SendRequest(ctx, "admin.getVMConfig", struct{}{}, res)
 	return res.Config, err
 }
+
+func (c *client) GetSignature(ctx context.Context, signatureRequest message.SignatureRequest) (*[bls.SignatureLen]byte, error) {
+	res := &message.SignatureResponse{}
+	err := c.requester.SendRequest(ctx, "snowman.getSignature", &signatureRequest, res)
+	return &res.Signature, err
```

Comment:
It seems like this is using the same client struct to hit a different API, but you would need to create two different clients one for the admin API and one in order to use the `GetSignature` API.

If we are creating a new client, I don't think it should be the case that one of the functions does not work and you have to initialize it differently.

Can we split this out into an admin client and a separate `GetSignature` or `Warp` API ?

Reply:
Can we also change this to `warp.getSignature` instead of `snowman` ?